docs/loading-in-profiles.md

```diff
@@ -21,16 +21,77 @@ uploadprofile /path/to/profile.js
 
 > `https://perf-html.io/from-url/{URL}`
 
-Profiles can also be loaded in from arbitrary URLs. In addition, you can then upload it to the online storage directly from perf.html's interface.
+Profiles can also be loaded in from arbitrary URLs. In addition, you can then upload it to the online storage directly from perf.html's interface. The profile needs to be served from https, and have the appropriate `Access-Control-Allow-Origin` header set to either `*` or `https://perf-html.io`.
```

Comment:
do you think we could update the CSP to accept urls like http://localhost ? Maybe specify a specific port ?

Reply:
I'll file a bug.

Reply:
Filed https://github.com/devtools-html/perf.html/issues/727